deploy/azure.tf

```diff
@@ -0,0 +1,192 @@
+# Terraform configuration file to deploy resouces to Azure.
+
+# Set up this variables in your environment before running Terraform:
+#
+#   export ARM_SUBSCRIPTION_ID=<YOUR SUBSCRIPTION ID>
+#   export ARM_CLIENT_ID=<SERVICE PRINCIPAL ID>
+#   export ARM_CLIENT_SECRET=<SERVICE PRINCIPAL SECRET>
+#   export ARM_TENANT_ID=<ACTIVE DIRECTORY DOMAIN ID>
+#
+# or set up them here in this configuration file:
+#
+#   provider "azurerm" {
+#     subscription_id = "xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+#     client_id       = "xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+#     client_secret   = "xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+#     tenant_id       = "xxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+#   }
+#
+# Then run this script with
+# ./terraform.exe plan -var-file=tfvars.json
+
+variable location {
+    type = "string"
+}
+
+variable "cosmosdb_failover_location" {
+    type = "string"
+}
+
+variable "azurerm_resource_group_00" {
+    type = "string"
+}
+
+variable "azurerm_storage_account_00" {
+    type = "string"
+} 
+
+variable "azurerm_storage_container_00" {
+    type = "string"
+}
+
+variable "azurerm_storage_queues" {
+    type    = "list"
+}
+
+variable "azurerm_cosmosdb_00" {
+    type = "string"
+}
+
+variable "azurerm_app_service_plan_00" {
+    type = "string"
+}
+
+variable "azurerm_application_insights_00" {
+    type = "string"
+}
+
+variable "azurerm_log_analytics_00" {
+    type = "string"    
+}
+
+# module "variables" {
+#     source = "./modules/variables"
+# }
+
+## RESOURCE GROUP
+
+# Create a resource group if it doesn’t exist
+resource "azurerm_resource_group" "azurerm_resource_group_00" {
+    name     = "${var.azurerm_resource_group_00}"
+    location = "${var.location}"
+    tags {
+        environment = "production"
+    }
+}
+
+## STORAGE
+
+resource "azurerm_storage_account" "azurerm_storage_account_00" {
```

Comment:
todo later: to easily configure multiple environments we could have all these resources moved in a module and include that module multiple times with different resource groups...

Reply:
as a a side note, this is already doable overriding the variable value in the command line or through environment.